pkg/crd/markers/crd.go

```diff
@@ -51,6 +52,9 @@ var CRDMarkers = []*definitionWithHelp{
 
 	must(markers.MakeDefinition("kubebuilder:deprecatedversion", markers.DescribesType, DeprecatedVersion{})).
 		WithHelp(DeprecatedVersion{}.Help()),
+
+	must(markers.MakeDefinition("kubebuilder:additionalmetadata", markers.DescribesType, AdditionalMetadata{})).
+		WithHelp(AdditionalMetadata{}.Help()),
```

Comment:
`Are we not adding annotations only?
What about the name here be kubebuilder:annotations

And we Annotations{}

Reply:
Ps is possible to add annotations outside of the metadata? 
Do we need to use this term here or is that redundant?

Reply:
Now it supports to add annotations and labels

```
// +kubebuilder:metadata:annotations=...
// +kubebuilder:metadata:labels=...
```

Not sure if there will be some other fields in metadata have to be added there. So we'd better put all of them in `kubebuilder:metadata`.